content/docs/0.8.x/operate/advanced_install_options/index.md

```diff
@@ -90,7 +90,7 @@ continuous-delivery.jmeterService.image.repository=YOUR_REGISTRY/keptn/jmeter-se
 <p>    
 ```console
 #!/bin/bash
-KEPTN_TAG=0.7.3
+KEPTN_TAG=0.8.0
 IMAGES_CONTROL_PLANE="centos/mongodb-36-centos7:1 nats:2.1.7-alpine3.11 connecteverything/nats-server-config-reloader:0.6.0 synadia/prometheus-nats-exporter:0.5.0 nginxinc/nginx-unprivileged:1.19.1-alpine keptn/remediation-service:${KEPTN_TAG} keptn/api:${KEPTN_TAG} keptn/bridge2:${KEPTN_TAG} keptn/eventbroker-go:${KEPTN_TAG} keptn/helm-service:${KEPTN_TAG} keptn/distributor:${KEPTN_TAG} keptn/shipyard-service:${KEPTN_TAG} keptn/configuration-service:${KEPTN_TAG} keptn/mongodb-datastore:${KEPTN_TAG} keptn/lighthouse-service:${KEPTN_TAG}"
```

Comment:
Pretty sure the images have to be updated as well.

Several images have been renamed (gatekeeper-service/approval-service?) and at least one new microservices has been added (shipyard-controller?)

Reply:
wow, thanks for pointing that out! I totally overlooked that. I have not aligned it with our Helm Charts.